systemtest/pom.xml

```diff
@@ -230,15 +230,22 @@
     <profiles>
 
         <profile>
-            <id>all</id>
+            <id>none</id>
             <properties>
-                <skipTests>false</skipTests>
+                <skipITs>true</skipITs>
```

Comment:
shouldn't be there `-skipTests`?

Reply:
No, we want to run the UTs as part of the regular UTs. But the ITs will run the STs -> which is what we don't want in regular build. LEts hope we will never have any actual ITs in the systemtests module ;-).